miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultSequencerReferenceService.java

```diff
@@ -38,7 +41,7 @@
 
   @Override
   public Collection<SequencerReference> listByPlatformType(PlatformType platformType) throws IOException {
-    return sequencerReferenceDao.listByPlatformType(platformType);
+    return sequencerReferenceDao.list(0, 0, true, "id", PaginationFilter.archived(false));
```

Comment:
Method name is `listByPlatformType`, but that's not what it's doing anymore? PlatformType parameter is being ignored